src/lib/running.ml

```diff
@@ -11,6 +11,11 @@ type t = { size   : int
          ; var    : float
          }
 
+type mean_update = ?size:int -> n_sum:float ->
+  n_sum_sq:float -> n_size:float -> t -> float -> float
+
+let default_mean_update ?size ~n_sum ~n_sum_sq ~n_size t v = n_sum /. n_size
+
```

Comment:
I just realized that this is probably not the most stable form for this (n_sum could overflow). It should probably be `t.mean + (v  - t.mean) / size_f`. I'll file an issue.